packages/website/styles.css

```diff
@@ -1,3 +1,40 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.load-wraper{
+    position: relative;
+    height: 400px;
+    background-color: rgb(38 38 38);
+    z-index: 1;
+    margin-top: .5em;
+    border-radius: .5rem;
+    overflow: hidden;
+  }
+
+.activity{
```

Comment:
so, in general with the website, it is preferred to use tailwind wherever possible. i am a little bit of a CSS noob, could you describe to me what these are aiming to achieve? we could maybe move this into either 1) inline tailwind css or 2) create a component, which uses the inline tailwind css or css modules (re-usable). let me know if you have any questions about this, happy to help.

Reply:
the load-wrapper and acticity class are resposible for the loading preview that shows while the posts are loading. The crop-image class is there to make sure only to show the upper half of the image. I'm not that good with css myself too and never worked with tailwind. But il will read up on tailwind and try to do it that way. If you have any suggestions please let me know.